### PR TITLE
ZD-734 Add projectId to Kibana logs

### PR DESCRIPTION
**Pull request by kdudina-smartling**

### CODE REVIEW

**Review by anastasiia-kryshtop — APPROVED** [review]

File: src/main/java/org/zendesk/client/v2/Zendesk.java [hunk 1]
```diff
@@ -3469,6 +3469,7 @@ public static class Builder {
         private String password = null;
         private String token = null;
         private String oauthToken = null;
+        private String projectId = null;
```
Comment (sl-pkryshtop) [inline src/main/java/org/zendesk/client/v2/Zendesk.java:3472]: Didn't get where projectId actually is set while constructing Zendesk object

File: src/main/java/org/zendesk/client/v2/AsyncHttpClientLogger.java [hunk 2]
```diff
@@ -12,17 +14,24 @@
...
     public static final String URL = "url";
     public static final String DIRECTION = "direction";
+    public static final String PROJECT_ID = "projectId";
+
     private final Logger logger;
+    private final ObjectMapper mapper;
+    private final String projectId;
 
-    public AsyncHttpClientLogger(Logger logger)
+    public AsyncHttpClientLogger(Logger logger, ObjectMapper mapper, String projectId)
```
Comment (sl-pkryshtop) [inline src/main/java/org/zendesk/client/v2/AsyncHttpClientLogger.java:30]: maybe just use MDC, if I am not wrong logstash adds params from MDC to log automatically

Comment (sl-pkryshtop) [reply src/main/java/org/zendesk/client/v2/AsyncHttpClientLogger.java:30]: sorry, logback

File: src/main/java/org/zendesk/client/v2/Zendesk.java (hunk 1 above)
Comment (sl-pkryshtop) [reply src/main/java/org/zendesk/client/v2/Zendesk.java:3472]: found it

File: src/main/java/org/zendesk/client/v2/AsyncHttpClientLogger.java (hunk 2 above)
Comment (kdudina-smartling) [reply src/main/java/org/zendesk/client/v2/AsyncHttpClientLogger.java:30]: We handle responses in async way, and MDC is per thread

**Pull request merged**